src/main/java/io/anserini/collection/WashingtonPostCollection.java

```diff
@@ -48,8 +48,8 @@
 
 /**
  * An instance of the <a href="https://trec.nist.gov/data/wapost/">TREC Washington Post Corpus</a>.
- * The collection contains 608,180 news articles and blog posts from January 2012 through August 2017,
```

Comment:
Should we reference stats for v2 and v3 separately? I believe v1 is defunct, right?

Reply:
We could mention that version 3 adds 154,418 new documents from 2018 and 2019 and removes (near) duplicates.

v2 was used in TREC News 2018 & 2019 and is basically v1 without exact duplicates.

Reply:
Yes, I think we should explicitly describe the date ranges and num of documents of v2 and v3.